bricks/template/__brick__/{{project_name.snakeCase()}}/ios/fastlane/Fastfile

```diff
@@ -99,14 +126,36 @@ platform :ios do
     )
   end
 
+  desc 'Build and upload Production app to App Store Connect'
+    lane :build_and_upload_production_app_store_connect_app do
+      set_app_version
+      bump_build
+      builder.build_app_store(
+        Constants.SCHEME_NAME_PRODUCTION,
+        Constants.PRODUCT_NAME_PRODUCTION,
+        Constants.BUNDLE_ID_PRODUCTION,
+        false
+      )
+      set_connect_api_key
+      upload_build_to_production_app_store_connect
+    end
+
+    desc 'upload production build to App Store Connect'
+    private_lane :upload_build_to_production_app_store_connect do
+      distribution_manager.upload_to_app_store_connect(
+        product_name: Constants.PRODUCT_NAME_PRODUCTION,
+        bundle_identifier: Constants.BUNDLE_ID_PRODUCTION
+      )
+    end
+
```

Comment:
@doannimble can we reformat the code to match the formatting of the other lines?
```suggestion
  desc 'Build and upload Production app to App Store Connect'
  lane :build_and_upload_production_app_store_connect_app do
    set_app_version
    bump_build
    builder.build_app_store(
      Constants.SCHEME_NAME_PRODUCTION,
      Constants.PRODUCT_NAME_PRODUCTION,
      Constants.BUNDLE_ID_PRODUCTION,
      false
    )
    set_connect_api_key
    upload_build_to_production_app_store_connect
  end

  desc 'upload production build to App Store Connect'
  private_lane :upload_build_to_production_app_store_connect do
    distribution_manager.upload_to_app_store_connect(
      product_name: Constants.PRODUCT_NAME_PRODUCTION,
      bundle_identifier: Constants.BUNDLE_ID_PRODUCTION
    )
  end

```